gravitee-apim-portal-webui-next/src/scss/material-design-overrides.scss

```diff
@@ -27,3 +27,7 @@ button {
 mat-card {
   --mdc-elevated-card-container-elevation: 0;
 }
+
+.mat-mdc-tab-header {
+  margin-bottom: 12px;
```

Comment:
Is this margin used by Audrey for all of the tabs? If so, I would recommend attaching it to `mat-tab-header` instead of the class (classes can change between Angular versions 🙃 ). If not, then the API details component can handle this spacing.

Reply:
The issue is that I can't access that from inside component, since mat-tab-header is child component of component I'm using so can't add class there. My ideas is to overwrite this by wrapping it inside specific class name e.g. `.api-details`

Reply:
Deleted